diagrams/stdlib-gen.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
```

Comment:
Is this file empty?

Reply:
Shouldn't be: I see it populated.

Reply:
You mean: there's only comments in there? I can remove these diagrams, as they're not useful, indeed.

Reply:
Yes, sorry, I meant that they seem to contain only comments, so I was not sure if there was something wrong or that was on purpose

Reply:
I'll remove them, as they serve no real purpose.